luma_core/src/vi.rs

```diff
@@ -5,20 +5,19 @@
 use crate::allocate::alloc_aligned;
 use crate::io::{read16, write16, write32};
 use alloc::boxed::Box;
-use core::pin::Pin;
 
 /// A struct representing the eXternal FrameBuffer, or XFB.  It represents the image that will be
 /// sent to the screen, in YUYV format.  It must be allocated as contiguous physical memory.
 pub struct Xfb {
-    data: Pin<Box<[u8]>>,
+    data: Box<[u16]>,
```

Comment:
The data needs to be pinned for the duration of the use of the XFB due to mmio usage. Rust doesn't confirm that the pointer wont move.

Reply:
Oh right, thanks, I’ve fixed that issue.